web-api/src/persistence/postgres/jobs/changeOfAddress/setChangeOfAddressCaseAsDone.ts

```diff
@@ -0,0 +1,12 @@
+import { pgUpdateTable } from '@web-api/persistence/postgres/utils/operation/pgUpdateTable';
+import { sql } from 'kysely';
+
+export async function setChangeOfAddressCaseAsDone(jobId: string) {
+  return await pgUpdateTable({
+    table: 'dwChangeOfAddress',
+    values: {
+      remaining: sql<number>`remaining - 1`,
+    },
+    where: qb => qb.where('jobId', '=', jobId),
```

Comment:
Is it worth having an index for jobId? (I don't know--I'm just calling out places we filter on a column without an index.)